js/pages/data-sources/components/reports/datadensity.js

```diff
@@ -52,7 +52,36 @@ define([
           yLabel: 'Concepts per Person'
         },
       };
-
+      
+      this.getData()
```

Comment:
Move getData to separate function and call it from constructor. Do the same with all similar cases

Reply:
it _is_ separate function (as a method of super class) and it is being called from constructor. Do you mean moving a callback which is parsing the data?